pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -8738,6 +8743,71 @@
+  middle_high_page_video_library_link: "View video library"
+  middle_high_page_cs_journeys_title: "Go Further with CS Journeys"
+  middle_high_page_cs_journeys_description: "Take the next step on your computer science journey. Explore careers, mentorship opportunities, online courses, scholarships, internships and more!"
+  middle_high_page_cs_journeys_link: "Explore CS Journeys"
+  middle_high_page_cs_title: "Computer Science in the Classroom"
+  middle_high_page_cs_description: "These courses should be led by a teacher! If you'd like to see them taught at your school, consider working with a teacher to bring them to your class."
+  middle_high_page_cs_card_title_1: "CS Discoveries"
+  middle_high_page_cs_card_title_2: "CS Principles"
+  middle_high_page_cs_card_title_3: "AP Computer Science A"
+  middle_high_page_cs_card_description_1: "Free set of elementary curricula that introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world."
+  middle_high_page_cs_card_description_2: "Free year-long curriculum that introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world. Can be taught as an introductory class and as an AP course."
+  middle_high_page_cs_card_description_3: "Introduce students to software engineering and object-oriented design while they learn the Java programming language in this free curriculum for AP® Computer Science A (AP® CSA)."
+  middle_high_page_cs_link_1: "Explore CS Dicoveries"
+  middle_high_page_cs_link_2: "Explore CS Principles"
+  middle_high_page_cs_link_3: "Explore CSA"
+
+  elemeentary_page_main_title: "Learning for Ages 5 to 11"
+  elementary_page_main_description: "Explore the world of computer science with engaging videos, self-paced tutorials, programming activities, and more!"
+  elementary_page_own_pace_title: "Continue Learning at Your Own Pace"
+  elementary_page_own_pace_description: "Create drawings, games, and more in these two excellent options for students getting started on their own learning the basics of computer science."
+  elementary_page_own_pace_card_title_1: "Pre-Reader Express Course"
+  elementary_page_own_pace_card_title_2: "Express Course"
+  elementary_page_own_pace_card_description_1: "Learn computer science at your own pace! Learn to create computer programs, develop problem-solving skills, and work through fun challenges! Make games and creative projects to share with friends, family, and teachers."
+  elementary_page_own_pace_card_description_2: "Learn computer science at your own pace! Learn to create computer programs, develop problem-solving skills, and work through fun challenges! Make games and creative projects to share with friends, family, and teachers."
```

Comment:
Nit: this looks like the same text in `elementary_page_own_pace_card_description_1`, you can use that string for both sections and remove this one